Sources/NIOSSH/Connection State Machine/Operations/AcceptsUserAuthMessages.swift

```diff
@@ -14,6 +14,14 @@
 
 protocol AcceptsUserAuthMessages {
     var userAuthStateMachine: UserAuthenticationStateMachine { get set }
+
+    var role: SSHConnectionRole { get }
+}
+
+/// This event indicates that server wants us to display the following message to the end user.
+public struct UserAuthBannerEvent: Hashable {
```

Comment:
For new types like this, we need to prefix them with the string `NIO`, to avoid type clashes.